src/workspaces/components/Tools.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {useSelector} from "react-redux";
+import {getTools} from "../selectors";
+import Button from "@material-ui/core/Button";
+
+type ToolProps = { workspaceUri: string }
+
+const Tools: React.FC<ToolProps> = (props: ToolProps) => {
+    const tools = useSelector(getTools);
+    const handleClick = (e : React.MouseEvent) => { e.stopPropagation() };
```

Comment:
I don't think that `onClick` prop on Button is needed if there is `href`

Reply:
There is onRowClick event for the whole row - without it, the row action redirects to the workspace before forwarding to the external URL, so there is a strange visual effect.